treeherder/etl/jobs.py

```diff
@@ -368,45 +368,51 @@ def _load_job(repository, job_datum, push_id, lower_tier_signatures):
                     'status': parse_status
                 })
 
-            _schedule_log_parsing(jl, result)
+            job_logs.append(jl)
+
+    _schedule_log_parsing(job, job_logs, result)
 
     return (job_guid, signature_hash)
 
 
-def _schedule_log_parsing(job_log, result):
+def _schedule_log_parsing(job, job_logs, result):
     """Kick off the initial task that parses the log data.
 
     log_data is a list of job log objects and the result for that job
     """
 
     # importing here to avoid an import loop
-    from treeherder.log_parser.tasks import parse_job_log
+    from treeherder.log_parser.tasks import parse_logs
 
     task_types = {
-        "errorsummary_json": ("store_failure_lines", "store_failure_lines"),
-        "buildbot_text": ("parse_log", "log_parser"),
-        "builds-4h": ("parse_log", "log_parser"),
+        "errorsummary_json",
+        "buildbot_text",
+        "builds-4h"
     }
 
-    # a log can be submitted already parsed.  So only schedule
-    # a parsing task if it's ``pending``
-    # the submitter is then responsible for submitting the
-    # text_log_summary artifact
-    if job_log.status != JobLog.PENDING:
-        return
+    job_log_ids = []
+    for job_log in job_logs:
+        # a log can be submitted already parsed.  So only schedule
+        # a parsing task if it's ``pending``
+        # the submitter is then responsible for submitting the
+        # text_log_summary artifact
+        if job_log.status != JobLog.PENDING:
```

Comment:
Hmm... I wonder if anyone actually ever uses this pre-parsed summary feature? Would be nice to rip this out in the future (probably not now).